controllers/schemas/user.js

```diff
@@ -114,5 +107,9 @@ module.exports.put = function (req, res, next) {
 }
 
 module.exports.getOne = function (req, res, next) {
-  validator.param(req, res, next, getOne)
+  validator.param(req, res, next, searchId.idSearch)
+}
+
+module.exports.delete = function (req, res, next) {
+  validator.param(req, res, next, searchId.idSearch)
```

Comment:
you are repeating code in getOne and delete, maybe you can create just one method with a distinct name ex. "verifyParamId" and it should work for both, do this in all the schemas

Reply:
aolved